cmake/Scripts/clang-check-test.sh.in

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This script performs a single analysis using clang-check
+# It is used by the 'make test' target in the buildsystems
+# Usually you should use 'ctest -C clang-check' rather than calling this script directly
+#
+# Parameters: $1 = Application binary
+#             $2 = Source file to process
+
+clangcheck_cmd=$1
+source_file=$2
+
+tmpfil=`mktemp`
```

Comment:
changed, old habits tend to stick ;) i also noticed this version of the script is an old one. i've updated to (what i consider) a prettier one.